server/app/controllers/applicant/ApplicantRoutes.java

```diff
@@ -49,4 +49,24 @@ public Call index(CiviFormProfile profile, long applicantId) {
       return controllers.applicant.routes.ApplicantProgramsController.index();
     }
   }
+
+  /**
+   * Returns the route corresponding to the applicant view action.
+   *
+   * @param profile - Profile corresponding to the logged-in user (applicant or TI).
+   * @param applicantId - ID of applicant for whom the action should be performed.
+   * @param programId - ID of program to view
+   * @return Route for the program view action
+   */
+  public Call view(CiviFormProfile profile, long applicantId, long programId) {
```

Comment:
**nitpick:** s/view/show to follow route naming convention

Reply:
Done